pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/instance/JavaInstanceRunnable.java

```diff
@@ -130,6 +130,8 @@
     // a read write lock for stats operations
     private ReadWriteLock statsLock = new ReentrantReadWriteLock();
 
+    private static final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Object mapper is not always thread safe (eg: when caching the reflection scan). We already have a thread-local utility to get an object mapper with the right configuration (`ObjectMapperFactory.getThreadLocal()`).

Reply:
👍